pontoon/teams/static/js/info.js

```diff
@@ -0,0 +1,46 @@
+/* global $ */
+$(function() {
+  var container = $('#main .container');
+
+  container.on('click', '.edit-info', function(e) {
+    e.preventDefault();
+    var infobox = $(".info");
```

Comment:
This selector (and the ones that follow) is too general. It's too dangerous too assume we'll never add elements to any of the other tabs or the page heading with such class name. Please use `container.find('.info')` instead.

And, try to use single quotes in selectors.